locations/hours.py

```diff
@@ -264,7 +263,7 @@
     "Ni": "Su",
     "Nie": "Su",
     "Niedz": "Su",
-    "Niedzela": "Su",
```

Comment:
I assume this was added because a website was giving us "Niedzela", if so, we are now losing Sunday hours from that spider.

Reply:
Hmm, when I looked at the diff here I thought it was a duplicate entry, not a misspelling. Must have been going too fast. Thanks for adding it back.